src/main/java/com/gdschongik/gdsc/global/exception/ErrorCode.java

```diff
@@ -56,6 +56,7 @@ public enum ErrorCode {
     UNIV_EMAIL_DOMAIN_MISMATCH(HttpStatus.BAD_REQUEST, "재학생 메일의 도메인이 맞지 않습니다."),
     MESSAGING_EXCEPTION(HttpStatus.BAD_REQUEST, "수신자 이메일이 올바르지 않습니다."),
     VERIFICATION_CODE_NOT_FOUND(HttpStatus.NOT_FOUND, "재학생 인증 코드가 존재하지 않습니다."),
+    EMAIL_NOT_SENT(HttpStatus.BAD_REQUEST, "이메일 인증 인증메일이 발송되지 않았습니다."),
```

Comment:
```suggestion
    EMAIL_NOT_SENT(HttpStatus.BAD_REQUEST, "재학생 인증 메일이 발송되지 않았습니다."),
```

++ UNIV_EMAIL로 시작하도록 enum 값 수정해주시면 좋을 것 같습니다